backend/app/services/device_monitor.py

```
@@ -0,0 +1,220 @@
+
+                # Emit error to task room
+                ws_emitter = get_ws_emitter()
+                if ws_emitter:
+                    try:
+                        await ws_emitter.emit_chat_error(
+                            task_id=subtask.task_id,
+                            subtask_id=subtask.id,
+                            error="Device connection lost",
+                            message_id=subtask.message_id,
+                        )
+                    except Exception as e:
```

Ruff (0.14.14)

Do not catch blind exception: `Exception`

(BLE001)